### PR TITLE
Added new module: Motivation

### LINKED ISSUE #78: Add new module: motivation
Module for showing motivational quotes and speeches.
@swapagarwal @TeAmP0is0N Please assign this to me.

### PR DESCRIPTION
**Pull request by adityabisoi**
Closes #78 

@swapagarwal @TeAmP0is0N Please review.

### CODE REVIEW

**Review by RohitRathore1 — CHANGES_REQUESTED** [review]
@adityabisoi Resolve conflicts.

**Review by RohitRathore1 — APPROVED** [review]

File: modules.py [hunk 1]
```diff
@@ -135,6 +135,16 @@ def reply(bot, message, intent, entities):
             bot.reply_to(message, data['definitions'][0]['definition'])
         else:
             bot.reply_to(message, data['message'])
+    elif intent == 'motivation':
+        url = "https://type.fit/api/quotes"
```
Comment (swapagarwal) [inline modules.py:139]: This API returns a lot of data. Need to find another one.

File: modules.py [hunk 2]
```diff
@@ -135,6 +135,16 @@ def reply(bot, message, intent, entities):
...
+        r=requests.get(url)
+        data=r.json()
+        quotes_list=[]
+        for i in data:
+            quotes_list.append(i)
+        i=random.randint(1,1643)
```
Comment (swapagarwal) [inline modules.py:145]: Magic number.

**Comment by swapagarwal** [comment]
Not the right fit for the bot.

**Pull request closed without merging**